plugins/rollbar/lib/samson_rollbar/rollbar_user_informer.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module SamsonRollbar
+  class RollbarUserInformer
+    ENV_KEY = "rollbar.exception_uuid"
+
+    class << self
+      attr_accessor :user_information_placeholder
+    end
+
+    def initialize(app)
+      @app = app
+    end
+
+    def call(env)
+      status, headers, body = @app.call(env)
+
+      if self.class.user_information_placeholder.present? && error_uuid = env[ENV_KEY]
```

Comment:
multiple parts using `self.class.user_information_placeholder` is kinda a smell ... 
ideally 
```
if (placeholder = self.class.user_information_placeholder.presence) && error_uuid = env[ENV_KEY]
  body = replace_placeholder(placeholder, replacement, body, headers)
```